### PR TITLE
Editor hotKeys

### PR DESCRIPTION
**Pull request by eyesplitter**
Added all hotkeys from #300 

All hotkeys work with `ctrl` key for the people who love windows

### CODE REVIEW

**Comment by Aivean** [comment]
@eyesplitter  Thank you for the contribution! The code looks good overall, however I found a bug when testing it locally. If a hotkey (say, `CMD-b`) is pressed when the form is not in focus it somehow gets "memorized" and triggered when the form gets focus and only the CMD key is pressed.

E.g. to reproduce:
1. Open reply textarea
2. Click outside it to make to lose focus
3. Press CMD-b
4. Click inside the textarea to make it gain focus
5. Press CMD  (just cmd)
6. Observe that `CMD-b` event is fired

Demo:
ezgif-1-abee070e90

Frankly I'm not sure what's going on, it maybe the bug in the `react-hotkeys-hook` library. Perhaps it would be easier to just extend the previous `handleKeyDown` approach?

File: frontend/package.json [hunk 1]
```diff
@@ -31,6 +31,7 @@
     "react-confirm-alert": "^3.0.6",
     "react-dom": "^18.0.0",
     "react-hook-form": "^7.29.0",
+    "react-hotkeys-hook": "^4.3.8",
```
Comment (4vanger) [inline frontend/package.json:34]: 7.4Kb gzipped minifed. Not too small
https://bundlephobia.com/package/react-hotkeys-hook@4.3.8

File: frontend/src/Components/CreateCommentComponent.tsx [hunk 2]
```diff
@@ -79,6 +96,7 @@ export default function CreateCommentComponent(props: CreateCommentProps) {
     const [isPosting, setPosting] = useState(false);
     const [previewing, setPreviewing] = useState<string | null>(null);
     const [mediaUploaderOpen, setMediaUploaderOpen] = useState(false);
+    const hotKeysRef = useHotkeys<HTMLTextAreaElement>(allowedKeys.join(','), (_, e) => handleHotKey(e), {enableOnFormTags: true});
```
Comment (4vanger) [inline frontend/src/Components/CreateCommentComponent.tsx]: Why global catch VS recommended pattern from documentation:
```
useHotkeys<HTMLTextAreaElement>('ctrl+enter,meta+enter', handleAnswer, {enableOnFormTags: true});
...
```
Afair it is more readable and extendable this way

Comment (Aivean) [reply frontend/src/Components/CreateCommentComponent.tsx]: I think it's related to the focus:
https://github.com/JohannesKlauss/react-hotkeys-hook#focus-trap

**Comment by eyesplitter** [comment]
@Aivean 
I suppose that was a bug in the `react-hotkeys-hook` module.
This behavior works only with the meta key.
So I switched the handler from `HotkeysEvent` to `KeyboardEvent` and now it works as expected.

File: frontend/src/Components/CreateCommentComponent.tsx (hunk 2 above)
Comment (eyesplitter) [reply frontend/src/Components/CreateCommentComponent.tsx]: > I think it's related to the focus: https://github.com/JohannesKlauss/react-hotkeys-hook#focus-trap

Yes, it is.

File: frontend/package.json (hunk 1 above)
Comment (eyesplitter) [reply frontend/package.json:34]: but the codebase looks better.

**Review by Aivean — APPROVED** [review]
LGTM

**Pull request merged**